app/frontend/src/locales/fr/translation.json

```diff
@@ -0,0 +1,131 @@
+        "uploadedFilesLabel": "Fichiers précédemment téléchargés:",
+        "noFilesUploaded": "Aucun fichier téléchargé pour le moment",
+        "loading": "Chargement...",
+        "manageFileUploads": "Gérer les téléchargements de fichiers",
+        "uploadingFiles": "Téléchargement de fichiers...",
+        "uploadedFileError": "Erreur lors du téléchargement du fichier - veuillez réessayer ou contacter l'administrateur.",
+        "deleteFile": "Supprimer le fichier",
+        "deletingFile": "Suppression du fichier...",
+        "errorDeleting": "Erreur lors de la suppression.",
+        "fileDeleted": "Fichier supprimé"
+    },
+    "developerSettings": "Paramètres développeur",
+
+    "chatEmptyStateTitle": "Discutez avec vos données",
+    "chatEmptyStateSubtitle": "Demandez n'importe quoi ou essayez un exemple",
+    "defaultExamples": {
+        "1": "Qu'est-ce qui est inclus dans mon plan Northwind Health Plus qui n'est pas dans le standard?",
+        "2": "Qu'arrive-t-il lors d'un examen de performance?",
+        "3": "Qu'est-ce qu'un Product Manager fait?",
+        "placeholder": "Tapez une nouvelle question (par exemple, mon plan couvre-t-il les examens oculaires annuels?)"
+    },
+    "askTitle": "Demandez à vos données",
+    "gpt4vExamples": {
+        "1": "Comparez l'impact des taux d'intérêt et du PIB sur les marchés financiers.",
+        "2": "Quelle est la tendance prévue pour l'indice S&P 500 au cours des cinq prochaines années? Comparez-le aux performances passées de l'S&P 500",
+        "3": "Pouvez-vous identifier une corrélation entre les prix du pétrole et les tendances du marché boursier?",
+        "placeholder": "Exemple: Mon plan couvre-t-il les examens oculaires annuels?"
+    },
+    "generatingAnswer": "génération de la réponse",
+
+    "tooltips": {
+        "submitQuestion": "Soumettre une question",
+        "askWithVoice": "Poser une question à l'aide de la voix",
+        "stopRecording": "Arrêter l'enregistrement de la question",
+        "showThoughtProcess": "Montrer le processus de réflexion",
+        "showSupportingContent": "Montrer le contenu de soutien",
+        "speakAnswer": "Parler réponse",
+        "info": "Info",
+        "save": "Sauvegarder"
+    },
+
+    "headerTexts":{
+        "thoughtProcess": "Processus de Réflexion",
+        "supportingContent": "Contenu d'Appui",
+        "citation": "Citation"
+    },
+
+    "labels": {
+        "headerText": "Configurer la génération de réponses",
+        "closeButton": "Fermer",
+        "promptTemplate": "Modèle de l'invite",
+        "temperature": "Température",
+        "minimumSearchScore": "Score de recherche minimum",
+        "minimumRerankerScore": "Score minimum du reranker",
+        "retrieveCount": "Récupérer ce nombre de résultats de recherche :",
+        "excludeCategory": "Exclure la catégorie",
+        "useSemanticRanker": "Utiliser le classement sémantique pour la récupération",
```

Comment:
```suggestion
        "useSemanticRanker": "Utiliser le reclasseur sémantique",
```

Reply:
Thank you, I have made the changes as per your suggestions.